deploy/helm/internal-charts/search-api-node/templates/service.yaml

```diff
@@ -8,7 +8,7 @@ spec:
     port: 80
     targetPort: 80
 {{- if .Values.global.exposeNodePorts }}
-    nodePort: 30122
```

Comment:
why do we want to change this?

Reply:
It was conflicting with the minion. Changed it back and changed the minion's nodePort instead